tritium-caffeine/src/main/java/com/palantir/tritium/metrics/caffeine/CaffeineCacheStats.java

```diff
@@ -48,10 +47,13 @@ private CaffeineCacheStats() {}
      * Callers should ensure that they have {@link Caffeine#recordStats() enabled stats recording}
      * {@code Caffeine.newBuilder().recordStats()} otherwise there are no cache metrics to register.
      *
+     * @deprecated use {@link #registerCache(TaggedMetricRegistry, Cache, String)}
+     *
      * @param registry metric registry
      * @param cache cache to instrument
      * @param name cache name
      */
+    @Deprecated
     public static void registerCache(MetricRegistry registry, Cache<?, ?> cache, String name) {
```

Comment:
We should probably deprecate the Guava cache version as well https://github.com/palantir/tritium/blob/aa6942273cce940270ddbc334f94c0e080dd3584/tritium-metrics/src/main/java/com/palantir/tritium/metrics/MetricRegistries.java#L227